### PR TITLE
feat: /spotify/*

### LINKED ISSUE #6601: Spotify RSS
### Website URL

https://www.spotify.com/

### Website description

Spotify is a digital music service that gives you access to millions of songs.

### What content should be included?

- [ ] Saved Tracks(Liked Songs): https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-tracks/
- [ ] Recent Top Tracks: https://developer.spotify.com/documentation/web-api/reference/personalization/get-users-top-artists-and-tracks/
- [ ] Playlist: https://developer.spotify.com/documentation/web-api/reference/playlists/add-tracks-to-playlist/
- [ ] Artist's Albums: https://developer.spotify.com/documentation/web-api/reference/artists/get-artists-albums/

### Additional description

Node.js SDK: https://github.com/thelinmichael/spotify-web-api-node

### PR DESCRIPTION
**Pull request by outloudvi**
## 该 PR 相关 Issue / Involved issue

fixes #6601

## 完整路由地址 / Example for the proposed route(s)

```routes
/spotify/playlist/4UBVy1LttvodwivPUuwJk2
/spotify/artist/31zCA3lKnObwtGBbDNnNAt
/spotify/saved/25
/spotify/top/tracks
/spotify/top/artists
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [x] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note

* Spotify 的 API 需要 token，因此 GHA 的测试应该会失败。生成这些 token 需要 clientId/clientSecret，但不一定需要用户登录。
* Spotify 专辑的 `release_date` 只能精确至日、月或年。 [src](https://developer.spotify.com/documentation/web-api/reference/#/operations/get-track)
* 对于与用户相关的端点，可能需要让用户填入 `refresh_token` 来获取。其它端点只需要 clientId/clientSecret。 [src](https://developer.spotify.com/documentation/general/guides/authorization/code-flow/)
* [Spotify Developer Terms](https://developer.spotify.com/terms/).

### CODE REVIEW

File: docs/install/README.md [hunk 1]
```diff
@@ -548,6 +548,16 @@ RSSHub 支持使用访问密钥 / 码，白名单和黑名单三种方式进行
...
+-   spotify 全部路由： [注册地址](https://developer.spotify.com)
+
+    -   `SPOTIFY_API_KEY`：Spotify 应用的 client ID
+
+    -   `SPOTIFY_API_SECRET`：Spotify 应用的 client secret
+
+-   spotify 用户相关路由
+
+    -   `SPOTIFY_REFRESHTOKEN`：用户在此 Spotify 应用的 refresh token。可以利用 [此 gist](https://gist.github.com/outloudvi/d1bbeb5e989db5385384a223a7263744) 获取。
+
```
Comment (TonyRL) [inline docs/install/README.md:560]: Please also update `docs/en/install/README.md`

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please provide screenshots of the xmls and feed content in RSS reader(that you preferred)

**Comment by outloudvi** [comment]
I've deployed <https://rsshub-8966.vercel.app/> with valid configurations, so please feel free to check:

* https://rsshub-8966.vercel.app/spotify/playlist/4UBVy1LttvodwivPUuwJk2
* https://rsshub-8966.vercel.app/spotify/artist/31zCA3lKnObwtGBbDNnNAt
* https://rsshub-8966.vercel.app/spotify/saved/25

Note that since the corresponding Spotify account is new and only for testing, Spotify hasn't curated Top Tracks and Top Artists for it. They may need several days or so to generate some contents:

* https://rsshub-8966.vercel.app/spotify/top/tracks
* https://rsshub-8966.vercel.app/spotify/top/artists

**Comment by outloudvi** [comment]
All the links on my test RSSHub instance are curated for now. Please feel free to check it out.

**Pull request merged**